app/views/stages/_fields.html.erb

```diff
@@ -116,14 +116,19 @@
 
   <div id="commands">
     <%= form.collection_check_boxes(:command_ids, form.object.all_commands, :id, :command) do |b| %>
+      <% command = b.object %>
+      <% others = (command.stages - [@stage]).map(&:name) %>
```

Comment:
fair enough. I don't think I've ever seen so much logic in a view before but OTOH its just four local variable definitions.